README.rst

```diff
@@ -1,2 +1,54 @@
 This package provides an ExplainCollection class
-that allows common MongoDB commands to be explained
\ No newline at end of file
+that allows that allows PyMongo's Collection methods to be explained.
+
+
+Tutorial
+########
+The intended use case for this package is to debug ``pymongo`` commands.
+This can be done by simply swapping out ``Collection`` for ``ExplainCollection``,
+which has the same methods but will run explain on them *instead* of executing them.
+The first step is to create a ``MongoClient`` instance::
+
+    client = MongoClient('mongodb://localhost:27017/')
```

Comment:
I would omit this step and directly with the next step. Client creation is something users must already be doing and is not something they need to do specially when wanting to use `pymongoexplain`.

Reply:
Done.